testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/batch/analyzer/transactiontimeout/TestWriter.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2016, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.as.test.integration.batch.analyzer.transactiontimeout;
+
+import javax.batch.api.chunk.AbstractItemWriter;
+import javax.inject.Named;
+import java.util.logging.Logger;
+
+@Named
+public class TestWriter extends AbstractItemWriter {
+    private static final Logger LOG = Logger.getLogger(TestWriter.class.getName());
+    public void writeItems(java.util.List<Object> items) throws Exception {
+        for(Object o : items){
+            LOG.info(o.toString());
```

Comment:
I think we should avoid logging these or log them as debug. CI servers don't always have a lot of disk space and logging unneeded data will increase usage of the disk space.